Silicon/Apple/AppleSiliconPkg/Drivers/AppleDartIoMmuDxe/AppleDartIoMmuDxe.c

```diff
@@ -40,13 +40,13 @@
 #include <Library/PcdLib.h>
 #include <Library/DxeServicesLib.h>
 #include <Library/TimerLib.h>
-#include <Include/libfdt.h>
+#include <Library/AppleDTLib.h>
 
 #include <Protocol/IoMmu.h>
 
 #include <Drivers/AppleDartIoMmuDxe.h>
 
-APPLE_DART_INFO DartInfo[FixedPcdGet32(PcdAppleNumDwc3Darts)];
+APPLE_DART_INFO DartInfo[FixedPcdGet32(PcdAppleNumDwc3Controllers) * 2];
```

Comment:
Just a note, while the library was initially made to bring up the USB-C DARTs, and this change is fine because it's a simplification, I never wanted the driver to only bring up the USB-C DARTs, I actually intended it to be used to manage all the DARTs on the system, however that proved to be more challenging than I had time for back when this code was made, so just pointing out that this code is likely getting a refactor down the line